tabula/file_util.py

```diff
@@ -45,14 +47,15 @@ def localize_file(path_or_buffer, user_agent=None, suffix=".pdf"):
         if ext != suffix:
             pid = os.getpid()
             filename = "{}{}".format(pid, suffix)
+        filename = os.path.join(temp_directory, filename)
```

Comment:
Rather adding `temp_directory` manually, how about using [tempfile.gettempdir()](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir)?

We should care about file path length handled in L44. For example, [macOS has a system limit of character size of a file name, i.e., 255 characters](https://discussions.apple.com/thread/148395)

You can check the example PDF in https://github.com/chezou/tabula-py/pull/196

Reply:
remove this line